frontend/public/kubevirt/components/nic.jsx

```diff
@@ -95,7 +95,7 @@ const NIC_TYPE_VM = 'nic-type-vm';
 const NIC_TYPE_CREATE = 'nic-type-create';
 
 export const NicRow = (onChange, onAccept, onCancel) => ({obj: nic}) => {
-  const namespace = getNamespace(nic.vm || nic.vmTemplate);
+  const namespace = getNamespace(nic.vmTemplate || nic.vm);
```

Comment:
can you please add the comment that order matters?

Reply:
\+ please add the comment to the props that the VM might be the template VM

Reply:
@suomiy not sure about the wording of the props comment, I think it's ok ?

Reply:
that one is okay. 

On the other hand, the `NOTE` is too verbose ^^. IMO `getNamespace(nic.vmTemplate || nic.vm) // order matters ` should be enough